docs/components/best-practices/operations/operating-camunda-c7.md

```diff
@@ -176,7 +176,7 @@ Sometimes, you may want an _emergency stop_ for a specific process instance or a
 
 For updating Camunda to a new version, follow the guide for [patch level updates](https://docs.camunda.org/manual/latest/update/patch-level/) or one of the dedicated [minor version update guides](https://docs.camunda.org/manual/latest/update/minor/) provided for each minor version release.
 
-A [rolling upgrade](https://docs.camunda.org/manual/latest/update/rolling-update) feature has been introduced in version 7.6. This allows users to update Camunda _without having to stop the system_. Outdated engine versions are able to continue to access an already updated database, allowing updates to clustered application servers one by one, without any downtime.
+A [rolling update](https://docs.camunda.org/manual/latest/update/rolling-update) feature has been introduced in version 7.6. This allows users to update Camunda _without having to stop the system_. Outdated engine versions are able to continue to access an already updated database, allowing updates to clustered application servers one by one, without any downtime.
```

Comment:
This makes me think the C7 docs may also inconsistently use upgrade vs. update, but I'm happy to see the URL says **update**.

Reply:
I can take a closer look at C7 docs and make adjustments there as well if you feel it's necessary in this case of terminology/how we refer to product upgrades.

Reply:
For consistency, take a look at how (or if) we use upgrade in C7 docs. 

This a great addition to the WCoE because as you mention in another comment, other people are having this same debate about upgrade vs. update.